models/user_bonus.ts

```diff
@@ -0,0 +1,67 @@
+import * as DocumentDb from "documentdb";
+import { DocumentDbModel } from "io-functions-commons/dist/src//utils/documentdb_model";
+import * as DocumentDbUtils from "io-functions-commons/dist/src/utils/documentdb";
+import * as t from "io-ts";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { FiscalCode, NonEmptyString } from "italia-ts-commons/lib/strings";
+import { tag } from "italia-ts-commons/lib/types";
+
+export const USER_BONUS_COLLECTION_NAME = "user-bonuses";
+export const USER_BONUS_MODEL_PK_FIELD = "fiscalCode";
+
+const UserBonus = t.interface({
+  // Id of the Bonus available to the user
+  bonusId: NonEmptyString,
```

Comment:
Cosmosdb by default sets a random generated unique id (named 'id') for each item. 

I'm thinking about providing our own id here (`<cf>-<bonusId>`) to speed up queries, 
in this case we have to add an `id` prop here.

`WHERE fiscalCode = x AND bonusId = Y `
will become
`WHERE id = x`

I'm not sure there's a real advantage in doing an equality check with the id (I'll try some example queries and check the RSUs)

cc @cloudify @francescopersico